packages/kotti-ui/source/kotti-row/index.ts

```diff
@@ -43,10 +44,12 @@ export const KtRow = attachMeta(
 						{
 							class: [
 								'kt-row',
-								props.justify !== 'start'
+								props.justify !== KottiRow.Justify.START
 									? `kt-row--justify-${props.justify}`
 									: '',
-								props.align !== 'top' ? `kt-row--align-${props.align}` : '',
```

Comment:
Same here

Reply:
KtRow uses non-zodded legacy types. Skip this until is refactored to use make-props.